src/locales/en-US/commands.json

```
@@ -497,5 +497,17 @@
   "aww": {
     "title": "Here is some eyebleach for ya!",
     "commandDescription": "Random image from /r/aww"
+  },
+  "mcquery": {
+    "commandDescription": "Fetch info from a MC:PC server.",
+    "commandUsage": "<address:port>",
+    "unknownServer": "Server is offline or doesn't exists.",
+    "noIP": "You need to give me a IP.",

```

If we're calling it "address", then we should use it everywhere.

```suggestion
    "noIP": "You need to give me an address.",
```